server/data/data-library/src/main/java/org/apache/james/rrt/lib/MappingsImpl.java

```diff
@@ -44,6 +43,10 @@
 
     private static final long serialVersionUID = 1L;
 
+    private static<T> Predicate<T> not(Predicate<T> p) {
+        return t -> !p.test(t);
+    }
```

Comment:
duplicated definition

Reply:
I think it is better to have a "!" in a lambda rather than a not...

Reply:
Deleted such as we can use .negative() from java 8